app/controllers/do_not_disturb_controller.rb

```diff
@@ -25,9 +25,8 @@ def create
   def destroy
     current_user.active_do_not_disturb_timings.destroy_all
     current_user.publish_do_not_disturb(ends_at: nil)
-    current_user.notifications.unprocessed.each do |notification|
-      NotificationEmailer.process_notification(notification, no_delay: true)
-    end
+    current_user.shelved_notifications.each(&:process)
```

Comment:
What happens if processing one of the shelved notification generates an exception? Do we want to `rescue` it like you do in the `process_shelved_notifications` job?

Reply:
I'm not sure what the difference of this is compared to any other code being called; I wrapped it in a `rescue` in the job, but this question could be asked about pretty much any code. I assume b/c it's tested that this call will not throw an exception.

I can add it if necessary, but I'm curious why you think it needs a `rescue`: whether it's just consistent with the job, or because this line is especially error-prone?

Reply:
I don't feel like we **need** it in the job either... 🤷

Reply:
I like being consistent. If the job is prone to throwing exception, this piece of could would be as well.

I'm 100% ditching it in the job if it's not needed 👍

Reply:
okay, I'll ditch it in the job!